static/js/hotkey.js

```diff
@@ -276,8 +277,7 @@ export function process_escape_key(e) {
 
         if (compose_state.composing()) {
             // Hide GIPHY popover if it's open.
-            if ($("#giphy_grid_in_popover").length) {
-                $("#compose_box_giphy_grid").popover("hide");
+            if (giphy.hide_giphy_popover()) {
                 $("#compose-textarea").trigger("focus");
                 return true;
```

Comment:
Is this logic correct?  If you have both the compose box and a message edit box open, which I think is possible, we should probably do this based on whichever the giphy popover was opened from.

Might be OK as a follow-up bug, since it may be pretty annoying to try to do the right thing for that case.

Reply:
Fixed in the commit.